app/build.gradle

```diff
@@ -200,6 +200,25 @@ android {
         }
     }
 
+    def shouldUseStaticVersionCode = { ->
+        if (gradle.hasProperty("userProperties.useStaticVersionCode")) {
+            return gradle."userProperties.useStaticVersionCode" == "true"
+        }
+        return false
+    }
+
+    if (shouldUseStaticVersionCode()) {
+        println("Build is being created with a static version code")
```

Comment:
Let's remove this, nobody is going to pay attention to it, and it will add noise to the build logs